File: 2053-kth-distinct-string-in-an-array/2053-kth-distinct-string-in-an-array.py
class Solution:
    def kthDistinct(self, arr: List[str], k: int) -> str:
        temp=[]
        for i in arr:
            c=arr.count(i)
            if c==1:
                if i not in temp:
                    temp.append(i)
        return (temp[k-1]) if len(temp)>=k else ""

File: 2053-kth-distinct-string-in-an-array/2053-kth-distinct-string-in-an-array.java
class Solution {
    public String kthDistinct(String[] arr, int k) {
        List<String> list = new ArrayList<>();
        for (int i = 0; i < arr.length;i++){
            boolean flag = true;
            for (int j = 0; j < arr.length;j++){
                if (arr[i].equals(arr[j]) && i != j){
                    flag = false;
                }
            }
            if (flag){
                list.add(arr[i]);
            }
        }
        return list.size() >= k ? list.get(k-1) : "";  
    }
}